README.md

```diff
@@ -4,98 +4,32 @@ In the website development timeline, we are currently evaluating the folllowing
 [Next.js](https://nextjs.org/) website (this repository), [MongoDB](https://www.mongodb.com/) database, and
 [Grafana](https://grafana.com/) dashboards.
 The easiest way to evaluate these potential solutions for our purposes is in [sailbot_workspace](https://github.com/UBCSailbot/sailbot_workspace).
-This repository starts from the [Next.js with-mongodb example](https://www.mongodb.com/developer/languages/javascript/nextjs-with-mongodb/)
-to connect to the MongoDB database in sailbot_workspace.
 
-## Example app using MongoDB
+## Database
 
 [MongoDB](https://www.mongodb.com/) is a general purpose, document-based, distributed database built for modern application
-developers and for the cloud era. This example will show you how to connect to and use MongoDB as your backend for your
-Next.js app.
+developers and for the cloud era. If you want to learn more about MongoDB, visit their docs site: [MongoDB Documentation](https://docs.mongodb.com/).
 
-If you want to learn more about MongoDB, visit the following pages:
+## Set Up
 
-- [MongoDB Atlas](https://mongodb.com/atlas)
-- [MongoDB Documentation](https://docs.mongodb.com/)
+1. Set each variable in `.env.local`:
+    - `MONGODB_URI` - Your MongoDB connection string: `mongodb://<ip-address-of-workspace-container>:27017/testdb`
+        - To find the ip address of the workspace container:
```

Comment:
Would it be better to use localhost and forward the port in the sailbot workspace docker files?

Reply:
O yes - that would make things a lot easier. Let me try that

Reply:
PR to port forward: https://github.com/UBCSailbot/sailbot_workspace/pull/170/files